bezier-rs/lib/src/lib.rs

```diff
@@ -171,28 +175,44 @@ impl Bezier {
 		}
 	}
 
+	///  Get the point on the curve based on the t-value provided, alias for compute
+	pub fn get(&self, t: f64) -> DVec2 {
+		self.compute(t)
+	}
+
+	/// Return a selection of equidistant points on the bezier curve
+	/// If no value is provided for `steps`, then the function will default `steps` to be 10
+	pub fn get_lookup_table(&self, steps: Option<i32>) -> Vec<DVec2> {
```

Comment:
`get` implies that we are just retrieving something rather than computing it - perhaps `compute_lookup_table` would be more clear?